ext/droption/droption.h

```diff
@@ -47,6 +47,7 @@
 #include <limits.h>
```

Comment:
For the PR title, let's try to shorten it:

How about: "i#5505 kernel trace: Support octal and hex values in droption"

We can move other details to the PR description.

Reply:
Done.